### ISSUE TITLE
Encryption

### ISSUE THREAD

**Issue by athiasjerome** [MAECProject/schemas#118]
Consider reuse of, or guidance to use, cyboxCommon:CipherEnum
Ref. http://cybox.mitre.org/language/version2.1/xsddocs/cybox_common_xsd.html#CipherEnum
for 
maecVocabs:CommonCapabilityPropertiesEnum-1.0
encryption algorithm

or as part of EncryptionType @encryption_mechanism
https://maecproject.github.io/data-model/4.1/ArtifactObj/EncryptionType

**Comment by ikiril01** [comment]
@athiasjerome good point, it would certainly make sense to align on these encryption algorithms between MAEC and CybOX. Perhaps we can make the MAEC CommonCapabilityPropertiesEnum a union of the CybOX CipherEnum and its own set of terms.
